src/register.js

```diff
@@ -0,0 +1,73 @@
+import videojs from 'video.js';
+import playMiddlewareFeature from './playMiddleware.js';
+
+const {playMiddleware, isMiddlewareMediatorSupported} = playMiddlewareFeature;
+
+// The plugin name.
+const NAME = 'ads';
```

Comment:
I recognize it's subjective, but extracting this to a variable doesn't seem valuable to me. The string is not likely to change and I don't see this as providing extra safety in this case. Importing it in other files seems tedious.